packages/lambda-xyz/src/cli/imagery/sentinel.ts

```diff
@@ -1,10 +1,74 @@
-import { MosaicCog } from './mosaic.cog';
 import { EPSG } from '@basemaps/geo';
+import { MosaicCog } from './mosaic.cog';
 
 MosaicCog.create({
-    id: '01E4PN40AEH10EH128AG28YSDM',
-    name: 'new_zealand_sentinel_2018-19_10m',
-    quadKeys: ['31'],
+    id: '01E6JDD1C6RMH29Q8Z97AEE79S',
+    name: 'sentinel',
+    quadKeys: [
+        '3113332',
```

Comment:
This is way too many quad keys for this imagery.

It should be at most about 5?

Reply:
Set max-cogs to 5, returned 3 quadkeys